sphinx/locale/__init__.py

```diff
@@ -51,8 +51,10 @@ def __repr__(self) -> str:
         try:
             return f'i{self.__str__()!r}'
         except Exception:
-            return (self.__class__.__name__
-                    + f'({self._catalogue}, {self._namespace}, {self._message})')
+            return (
```

Comment:
I'd suggest we do some PRs where we replace '+' with f-strings whenever possible instead of mixing the two.